xml/docu_styleguide.language.xml

```diff
@@ -422,16 +422,26 @@
   </listitem>
   <listitem>
   <para>
+  Use <tag class="emptytag">emphasis</tag> to call attention to new words or phrases, 
+  for example, <quote>using so-called <emphasis>target units</emphasis>,</quote>
+  to use words in a non-standard way, for example, <quote>packages can get in an
+  <emphasis>orphaned</emphasis> state,</quote> and to refer to a word or term itself,
+  for example, <quote>The word <emphasis>processor</emphasis> came into use around 1910.</quote>
+  </para>
+  </listitem>
+  <listitem>
+  <para>
   Do not use quotation marks to indicate irony.
   Avoid irony in technical writing.
   See also <xref linkend="sec-tone"/>.
   </para>
-  </listitem>
+  </listitem>  
   </itemizedlist>
   <para>
   To create quotations, use the <tag class="emptytag">quote</tag>
   element. This element is easier to localize than hardcoded quotation marks.
   During processing, localized quotation marks are added automatically.
+  Avoid using single quotation marks.
```

Comment:
Is this a thing? If we use the `<quote>` tag, this won't happen. Sounds a bit unnecessary.

Reply:
I think we need to discourage the use of single quote marks like 'this'. We mention that hardcoded, i.e. hand-written double marks should not be used, so I believe it's worth mentioning the second type, too.

Reply:
Ok, yes, that's sounds reasonable. Then keep it. :slightly_smiling_face: